.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
+- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
+- [ ] Gallery example in `./doc/examples` (new features only)
+- [ ] Unit tests
+
+[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]
+
+## References
```

Comment:
I see at least 2 reasons to keep this section: (a) For bugfixes and enhancements: helps to manage related issues, (b) For new features (with or without corresponding issues): asks for information on the algorithm, which is helpful for profitability evaluation.
This is more like a service section and it is least important (so, placed at the end of template and starts with `If known...`).